wifiphisher/common/interfaces.py

```diff
@@ -1,163 +1,147 @@
-#pylint: skip-file
 """
-This module was made to handle all the interface related operations for
-Wifiphisher.py
+This module was made to handle all the interface related operations of
+the program
 """
 
 import pyric
 import pyric.pyw as pyw
 
 
-class NotEnoughInterfacesFoundError(Exception):
-    """
-    Exception class to raise in case of a finding less than enough interfaces.
-    """
```

Comment:
I think this will raise confusion. If a user has one card that supports both AP and Monitor modes, he will see an error that he needs an interface that supports Monitor Mode. It won't be clear to the user that he needs a separate card for Monitor Mode.

We need to be clear when raising an exception. Not having enough adapters is different from having the required number of adapters but they are not supporting the right modes. Otherwise, people will start opening issues about what is required again.